spec/rubocop/cop/style/and_or_spec.rb

```diff
@@ -373,6 +373,18 @@ def y
       end
     end
 
+    context 'when left hand side is a comparison method' do
+      # Regression: https://github.com/bbatsov/rubocop/issues/4451
+      it 'does not autocorrect' do
```

Comment:
Whoops. My first, naïve approach was to just ignore these cases. Good catch! 🙂 

Fixed this and rebased.